toolkit/tools/pkg/imagecustomizerlib/customizepackages.go

```diff
@@ -178,18 +163,30 @@ func installOrUpdatePackages(action string, allPackagesToAdd []string, imageChro
 	return nil
 }
 
-// Process the stdout of a `tdnf install -v` call and send the list of installed packages to the debug log.
-func tdnfInstallOrUpdateStdoutFilter(args ...interface{}) {
-	const tdnfInstallPrefix = "Installing/Updating: "
+func callTdnf(tnfArgs []string, tdnfMessagePrefix string, imageChroot *safechroot.Chroot) error {
+	seenTransactionErrorMessage := false
```

Comment:
we can deduce the tdnfMessagePrefix based on `tnfArgs` install/remove values... Would probably be nicer to eliminate this parameter.

Reply:
IMHO, that would add a bunch of code for no real benefit.